tests/installation/welcome.pm

```diff
@@ -113,7 +113,8 @@ sub run {
 
     # Verify install arguments passed by bootloader
     # Linuxrc writes its settings in /etc/install.inf
-    if (get_var('NETBOOT') && !check_var('BACKEND', 'ipmi') && get_var('INSTALL_SOURCE')) {
+    if (get_var('NETBOOT') && !check_var('BACKEND', 'ipmi') &&
+        get_var('INSTALL_SOURCE') && check_var('ARCH', 'x86_64')) {
```

Comment:
As it's already second patch and condition is kind of complex, tbh I would opt to some special variable which enables this check and we set it in affected test suites. WDYT?

Reply:
I was thinking about it, but on the other hand do we want to introduce another test variable, when the test should be executed only if several certain conditions should be met ? If it's going to improve readability of the test suite within openqa test definitions and the code on the same time, then why not ? I do not have any problems if we choose to go one way or the other. :)

Reply:
I would actually just use `is_remote_backend` at least to express this logic. With test variable we have better control where to run this check.

Reply:
Updated, I am running VR locally